search/search-client-elasticsearch/src/main/resources/camunda-web-session.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
❓ Q: Shouldn't this be part of the webapps-schema module ?

https://github.com/camunda/camunda/tree/main/webapps-schema/src/main/resources/schema/elasticsearch/create/index